doc/advanced-configuration/Services.md

```diff
@@ -1,80 +1,94 @@
+#### `services.service_mongoose_system_metrics.report`
+* **Syntax:** boolean
+* **Default:** `false`
+* **Example:** `report = true`
+
+Explicit acknowledgement that the metrics are gathered and reported.
+Enabling this option silences the notification reminder that metrics are gathered.
+#### `services.service_mongoose_system_metrics.no_report`
+* **Syntax:** boolean
+* **Default:** `false`
+* **Example:** `no_report = true`
+
+When this option is set, System Metrics Service is not started and metrics are not collected.
 Having this option enabled, stops the notification warning that the functionality is not being used.
-* `intial_report`:
-    * **Description:** Time delay counted when the service is started after which the first metrics report is created and sent.
-    * **Syntax:** `{initial_report, Delay}`
-    * **Default:** 300000ms (5min).
-    * **Example:** `{intial_report, 300000}`
-* `periodic_report`:
-    * **Description:** Time delay for a periodic update report to be created and sent.
-    * **Syntax:**`{periodic_report, Delay}`
-    * **Default:** 108000000ms (3h)
-    * **Example:** `{periodic_report, 108000000}`
-* `tracking_id`:
-    * **Description:** Tracking ID to forward the reported metrics so that they can be viewed in the Google Analytics dashboard.
-    * **Syntax:**`{tracking_id, TrackingID}`
-    * **Default:** disabled
-    * **Example:** `{tracking_id, UA-123456789}`
-
-Removing the `service_mongoose_system_metrics` entry from list of services will result in the service not being started.
+
+#### `services.service_mongoose_system_metrics.intial_report`
+* **Syntax:** non-negative integer
+* **Default:** `300000` (milliseconds - 5 minutes).
+* **Example:** `intial_report = 300000`
+
+Time delay counted when the service is started after which the first metrics report is created and sent.
+
+#### `services.service_mongoose_system_metrics.periodic_report`
+* **Syntax:** non-negative integer
+* **Default:** `108000000` (milliseconds - 3 hours)
+* **Example:** `periodic_report = 108000000`
+
+Time delay for a periodic update report to be created and sent.
+
+#### `services.service_mongoose_system_metrics.tracking_id`:
+* **Syntax:** string
+* **Default:** no default.
+* **Example:** `tracking_id = "UA-123456789"`
+
+Tracking ID to forward the reported metrics so that they can be viewed in the Google Analytics dashboard.
+
+Removing the `services.service_mongoose_system_metrics` entry will result in the service not being started.
 Metrics will not be collected and shared.
 It will generate a notification that the feature is not being used.
 The notification can be silenced by setting the `no_report` option explicitly.
-
-#### Example configuration
-```
-{service_mongoose_system_metrics, [
-                                   report,
-                                   {intial_report, 300000},
-                                   {periodic_report, 108000000}
-                                  ]
-}
-```
```

Comment:
I was thinking about it, but it would double the configuration on line ~15 a little bit. Could the part with `service_mongoose_system_metrics` just be added there to show the full configuration?

Reply:
Yes, I think we could add full config with both of them but maybe at the bottom of the file.